src/secp256k1/src/cpp/Scalar.cpp

```diff
@@ -269,10 +269,10 @@ unsigned char* Scalar::deserialize(unsigned char* buffer) {
 
 std::string Scalar::GetHex() const {
     unsigned char buffer[32];
-    std::stringstream ss;
-
     serialize(buffer);
 
+    std::stringstream ss;
+    ss << std::hex;
     for (int i = 0; i < 32; ++i) {
```

Comment:
Change hard-coded size to size of `buffer` will be better. My idea is change `buffer` to `std::array` will be better so we can foreach on it or get it size easily.

Reply:
fixed